package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "run-shell",
```

Comment:
you named the repo runshell.  please make it consistent, either rename the repo or change here